### DISCUSSION TITLE
sorry can some body help with last project of course 1, i writing some thing and make submit can  i came back and writing again?

### DISCUSSION THREAD

**Discussion by MitrofanovDmitry** [OpenMined/courses#234] [Project: Privacy Product Specification] [answered]
sorry can some body help with last project of course 1, i writing some thing and make submit can  i came back and writing again?

**Comment by yemikifouly** [comment, marked as answer]
Hello! 👋 Yes! No worries! There is an auto-saving feature that automatically saves your work and allows you to go away and come back to finish. 🙂 Now, if you wish, for additional security, you can copy and paste your work in a google document that you can save. I hope this was helpful. 🙂
